packages/storybook/src/stories/Header/Header.stories.ts

```diff
@@ -0,0 +1,25 @@
+import { Header, HeaderProps } from '../../../../frontend-design-poc/src/components/Header/Header';
```

Comment:
Could we figure out how to not do relative imports between packages before we have a ton of these?